src/v0/destinations/marketo_bulk_upload/util.js

```diff
@@ -142,54 +142,48 @@ const getAccessTokenURL = (config) => {
 
 // Fetch access token from client id and client secret
 // DOC: https://developers.marketo.com/rest-api/authentication/
-const getAccessToken = async (config) =>
-  authCache.get(getAccessTokenCacheKey(config), async () => {
-    const url = getAccessTokenURL(config);
-    const { processedResponse: accessTokenResponse } = await handleHttpRequest('get', url, {
-      destType: 'marketo_bulk_upload',
-      feature: 'transformation',
-    });
+const getAccessToken = async (config) => {
```

Comment:
Can you please write tests for all the scenarios in this function?

Reply:
added